package.json

```diff
@@ -62,7 +62,7 @@
     "react-test-renderer": "16.8.0",
     "react-transition-group": "^2.5.3",
     "style-loader": "^0.23.1",
-    "stylelint": "^10.1.0",
+    "stylelint": "^13.3.3",
```

Comment:
What version of node is required to support this version of `stylelint`?

Reply:
There isn't a node version specified in their package.json

Is this a concern? It seems to be working for our dev environments.

Reply:
Just trying to keep aware. I think CI is ~node10. We should probably push to standardize node version in bigmaven. 😅

Reply:
Yeah, I agree! 

We know if it runs in MDS then it is Node 10. If it runs in bigmaven then it is Node 8. 

Stylelint it pretty popular so I'm going to assume it is safe to run LTS version of Node for it -- and that is usually what _we_ should be running in production environments.